packages/firestore/src/lite-api/aggregate.ts

```diff
@@ -15,72 +15,114 @@
  * limitations under the License.
  */
 
-import { Value } from '../protos/firestore_proto_api';
-import { invokeRunAggregationQueryRpc } from '../remote/datastore';
 import { hardAssert } from '../util/assert';
 import { cast } from '../util/input_validation';
+import { invokeRunAggregationQueryRpc } from '../remote/datastore';
+import { Value } from '../protos/firestore_proto_api';
 
-import { getDatastore } from './components';
+import { DocumentFieldValue, Query, queryEqual } from './reference';
 import { Firestore } from './database';
-import { Query, queryEqual } from './reference';
+import { getDatastore } from './components';
 import { LiteUserDataWriter } from './reference_impl';
 
+import { deepEqual } from '@firebase/util';
+import { FieldPath } from './field_path';
+
 /**
- * An `AggregateQuery` computes some aggregation statistics from the result set of
- * a base `Query`.
+ * An `AggregateField` computes some aggregation statistics from the result set of
+ * an aggregation query.
  */
-export class AggregateQuery {
-  readonly type = 'AggregateQuery';
-  /**
-   * The query on which you called `countQuery` in order to get this `AggregateQuery`.
-   */
-  readonly query: Query<unknown>;
+export class AggregateField<T> {
+  type = 'AggregateField';
+  _datum?: T;
 
-  /** @hideconstructor */
-  constructor(query: Query<unknown>) {
-    this.query = query;
-  }
+  constructor(readonly subType: string) {}
```

Comment:
added subTyoe to differentiate which function created the AggregateField (min(), count(), sum()....)